portfolio/src/main/webapp/script.js

```diff
@@ -63,3 +63,9 @@ function grow() {
     }
   }
 }
+
+function getGreeting() {
+  fetch('/data').then(response => response.text()).then(greeting => {
+    document.getElementById('greeting-container').innerHTML = greeting;
```

Comment:
This is ok as long as you control the greetings from the server. But keep in mind that in general it is not safe to put arbitrary text into innerHTML. https://developer.mozilla.org/en-US/docs/Web/API/Element/innerHTML#Security_considerations

Reply:
I see I see! Thanks for the suggestion.